migrations/V98__Add_Cedar_System_Id_To_Intake_and_508_Request_Tables.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE system_intakes ADD COLUMN cedar_system_id text;
```

Comment:
This file name could conflict with the migration in https://github.com/CMSgov/easi-app/pull/1440; might want to rename this one V99.

Reply:
Will probably be clearer now that it's merged to master, as well!